_sass/_uswds-custom-styles.scss

```diff
@@ -162,3 +209,24 @@
     text-align: center;
   }
 }
+
+// ==========================================================================
+// Footer
+// ==========================================================================
+
+// Additional social networks
+.usa-social-link--flickr {
+  @include add-background-svg("social-icons/flickr", $custom-image-dir);
+}
+
+.usa-social-link--instagram {
+  @include add-background-svg("social-icons/instagram", $custom-image-dir);
+}
+
+.usa-social-link--linkedin {
+  @include add-background-svg("social-icons/linkedin", $custom-image-dir);
+}
+
+.usa-social-link--email {
+  @include add-background-svg("social-icons/email", $custom-image-dir);
+}
```

Comment:
7. Added additional social icons to footer. Created `$custom-image-dir` variable to be able to update the custom image path once.

---

### We've reached the end 🥳

To summarize we've:
1. Created a microsite landing page re-using existing components
1. Updated our base colors
1. Checked the accessibility of our components based on color changes and made updates accordingly
1. Themed our global figcaption element
1. Added additional social icons to footer